tests/unit/layout-test.js

```diff
@@ -126,7 +125,7 @@ describe('layout', function () {
   })
 
   // FIXME: get test passing
```

Comment:
Did you get this test passing or is this from your automated linting tools?  If you did we can remove this comment.